x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/view/index.test.tsx

```diff
@@ -1037,14 +1071,18 @@ describe('when on the endpoint list page', () => {
       agentId = hosts[0].metadata.elastic.agent.id;
 
       setEndpointListApiMockImplementation(coreStart.http, {
-        endpointsResults: [hostInfo],
+        endpointsResults: hostInfo,
         endpointPackagePolicies: [packagePolicy],
         agentPolicy,
       });
     };
 
     beforeEach(async () => {
       mockEndpointListApi();
+      (useUserPrivileges as jest.Mock).mockReturnValue({
+        ...mockInitialUserPrivilegesState(),
+        endpointPrivileges: { canAccessResponseConsole: true },
```

Comment:
I thought `true` is already the default value, no?

---

(optional) If you still need it, then I would suggestion: include all of the endpoint privileges and only overwrite the one you need:

```typescript
endpointPrivileges: {
  ...mockInitialUserPrivilegesState().endpointPrivileges,
  canAccessResponseConsole: true
}
```

Reply:
It was not which was why I had to add this.